package.json

```diff
@@ -50,7 +51,7 @@
     "release:patch": "yarn build && standard-version --release-as patch",
     "release:alpha": "yarn build && standard-version --prerelease alpha",
     "start": "yarn build:tokens && yarn storybook",
-    "storybook": "start-storybook -p 9009 -s public",
```

Comment:
`-s` stands for `--static-dir` -- I don't believe we're serving any static assets (besides the fonts, which seem to already be handled), and the `public/` folder doesn't even exist